source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,328 @@
+commit the migration to make it permanent or revert the migration to return your
+App to Partiton-Based Sync.
+
+If you don't commit or revert the migration, it will automatically commit
+according to your 
+:atlas:`minimum oplog window </atlas/cluster-additional-settings/#set-minimum-oplog-window>`. 
+For example, if your minimum oplog window is 48 hours, you will have 48 hours to
+revert your migration.
+
+After a migration is committed, the old Partition-Based Sync metadata is deleted
+and you cannot revert the migration.
+
+**To manually commit a migration:** 
+
+#. In the App Services UI, go to your App and select the Device Sync page.
+#. At the top of the page, select the three dots next to 
+   :guilabel:`Manage Sync Migration`.
+#. From the list of options, select :guilabel:`Commit migration to Flexible Sync`.
+   
+   .. image:: /images/commit-migration.png
+      :alt: Commit a migration
+#. Confirm migration commit by typing "Commit", then select :guilabel:`Commit`.
+   
+   .. image:: /images/confirm-migration-commit.png
+      :alt: Confirm migration commit
+   
+   This will commit your migration and you will not be able to recover your
+   Partition-Based Sync Metadata.
+
+.. _realm-sync-migrate-revert:
+
+Revert Migration
+~~~~~~~~~~~~~~~~
+
+You can revert a migration after it has completed and before it has been committed.
+
+Completed migrations are automatically committed in accordance with your 
+:atlas:`minimum oplog window </atlas/cluster-additional-settings/#set-minimum-oplog-window>`.
+You can increase or decrease the amount of time you have to 
+revert a migration by adjusting the minimum oplog window.
+
+You cannot revert a migration that has been committed.
+
+Similar to canceling a migration, if you revert a migration, you need to 
+start over from the beginning with future migration attempts.
+
+**To revert a migration:** 
+
+#. In the App Services UI, go to your App and select the Device Sync page.
+#. At the top of the page, select the three dots next to 
+   :guilabel:`Manage Sync Migration`.
+#. From the list of options, select 
+   :guilabel:`Revert back to Partition-Based Sync`.
+   
+   .. image:: /images/revert-migration.png
+      :alt: Revert a migration
+   
+   This will revert your migration. You will need to migrate again if you 
+   want to enable Flexible Sync in the future.
+
+.. _realm-sync-migrate-client:
+
+Migrate Client App to Flexible Sync
+-----------------------------------
+
+After your app's backend has been migrated to Flexible Sync, we recommend
+that you also migrate your client app to Flexible Sync. When you migrate your
+client app to Flexible Sync, the client will stop automatically creating
+sync subscriptions for each Realm object model in your app's data model.
+
+We also recommend removing all sync subscriptions and then creating new subscriptions
+for your data. This is the clearest way to sync data after a migration.
```

Comment:
hmm, not sure i get what this second sentence means. IIUC, maybe "This is the clearest way to control what data is synced after a migration" ?

Reply:
Yes, thank you for pointing out the ambiguity here.